yarn-project/circuits.js/src/structs/kernel/__snapshots__/index.test.ts.snap

```diff
@@ -1306,7 +1306,7 @@ sibling_path: [ 0x1075 0x1076 0x1077 0x1078 0x1079 0x107a 0x107b 0x107c 0x107d 0
 is_transient: 0
 hint_to_commitment: 0x0
  ]
-portal_contract_address: 0x4141414141414141414141414141414141414141
```

Comment:
Hmm, is this expected?

Reply:
Well the type was changed to Fr from ETHAdress. The latter type has factory enforcing 20-byte length afaik. While now we are using  a Fr factory which is just incrementing a seed.

Reply:
I pushed a new commit preserving this snapshot.